ladybug_comfort/cli/utci.py

```diff
@@ -0,0 +1,364 @@
+# coding=utf-8
+"""Utility functions for calculating Universal Thermal Climate Index (UTCI).
+
+UTCI is a thermal comfort model strictly for the outdoors. It is an international
+standard for outdoor temperature sensation (aka. "feels-like" temperature) and
+is one of the most common of such "feels-like" temperatures used by
+meteorologists. UTCI that attempts to satisfy the following requirements:
+
+1) Thermo-physiological significance in the whole range of heat
+    exchange conditions of existing thermal environments
+2) Valid in all climates, seasons, and scales
+3) Useful for key applications in human biometeorology.
+
+While UTCI is designed to be valid in all climates and seasons, it assumes
+that human subjects are walking (with a metabolic rate around 2.4 met) and
+that they naturally adapt their clothing with the outdoor temperature.
+For outdoor situations that do not fit these criteria, the Physiological
+Equivalent Temperature (PET) model is recommended.
+"""
+import numpy as np
+
+
+def universal_thermal_climate_index(ta, tr, vel, rh):
```

Comment:
To help prevent conflicts if someone imports both this function and the base function, maybe this one should be called `universal_thermal_climate_index_np`

Reply:
I renamed the function.